stac_mjx/main.py

```diff
@@ -1,47 +1,53 @@
-"""This module is the entry point for the stac-mjx algorithm."""
-
 import mujoco
 import jax
-from jax import numpy as jnp
-from jax.lib import xla_bridge
+
 from dm_control import mjcf
-import numpy as np
 
-import os
 import pickle
 import time
-import argparse
-import random
 import logging
-import sys
-import hydra
-from hydra import initialize
 from omegaconf import DictConfig, OmegaConf
 
-import utils
-import controller as ctrl
+from stac_mjx import utils
+from stac_mjx import controller as ctrl
+
+
+def load_configs(stac_config_path: str, model_config_path: str) -> DictConfig:
+    """Initializes configs.
+
+    Args:
+        stac_config_path (str): path to stac yaml file
+        model_config_path (str): path to model yaml file
+
+    Returns:
+        DictConfig: stac.yaml config to use in run_stac()
+    """
+    utils.init_params(
+        OmegaConf.to_container(OmegaConf.load(model_config_path), resolve=True)
+    )
+    return OmegaConf.load(stac_config_path)
```

Comment:
[Google style guide](https://google.github.io/styleguide/pyguide.html#24-exceptions) suggests to use exceptions only when we need to break out of control flow, which doesn't seem necessary in this case (it will crash anyway with an appropriate message). And it looks like [sleap-io](https://github.com/talmolab/sleap-io/blob/main/sleap_io/io/nwb.py#L40) doesn't have exception handling for bad paths. So I'll ignore this suggestion.